src/resources/elements/linkfield.js

```diff
@@ -45,6 +45,21 @@ export class Linkfield extends Field {
     return field;
   }
 
+  resolvePath(path) {
```

Comment:
Just nearby. Generally, the comment(s) should live near the code, whether it is a block of code or a single line.